tests/cases/dom/manipulate/createElement.js

```diff
@@ -0,0 +1,157 @@
+import QUnit from "qunitjs";
+import {createElement} from "../../../../dom/manipulate";
+
+QUnit.module("dom/manipulate/createElement()",
+    {
+        beforeEach: () =>
+        {
+            document.getElementById("qunit-fixture").innerHTML = "";
+        },
+    }
+);
```

Comment:
Straight from the QUnit docs:

> By putting it in the `#qunit-fixture` element, we don’t have to worry about DOM changes from one test affecting other tests, because QUnit will automatically reset the markup after each test.

http://qunitjs.com/intro/#testing-the-dom-manipulation